src/box/box.cc

```diff
@@ -2834,6 +2835,28 @@ box_session_push(const char *data, const char *data_end)
 	return rc;
 }
 
+API_EXPORT int
+box_bloom_bsize(uint32_t space_id, uint32_t index_id, size_t *result) {
+	struct space *space;
+	struct index *index;
+
+	/** See check_index() for explanation. */
+	space = space_cache_find(space_id);
+	if (space == NULL)
+		return -1;
+	if (access_check_space(space, PRIV_R) != 0)
+		return -1;
+	index = index_find(space, index_id);
+	if (index == NULL)
+		return -1;
+	struct vy_lsm *lsm = vy_lsm(index);
+	ssize_t bsize = lsm->bloom_size;
+	if (bsize < 0)
+		return -1;
```

Comment:
This logic should be moved to `vy_lsm.{h, c}`

Reply:
Am I right that I should move only the logic of getting bloom_size field from lsm so it looks like `ssize_t bsize = vy_lsm_bloom_size(lsm)`?

Reply:
1.) Moved index to lsm casting in vy_lsm_bloom_size()
2.) Shouldn't we return -1 in case of en error?

Reply:
No, I mean how can bloom_size be possibly negative?

Reply:
For some reason I thought that returned `bloom_size` has `ssize_t` type and decided to check if it's equals to -1.
Deleted if statement.